README.rst

```diff
@@ -41,3 +41,32 @@ Commercial support is also available by `Scrapinghub`_.
 .. _create a question issue: https://github.com/scrapinghub/splash/issues/new?labels=question
 .. _Scrapinghub: https://scrapinghub.com
 .. _scrapy-splash library: https://github.com/scrapy-plugins/scrapy-splash
+
+Building qtwebkit from sources
```

Comment:
Could you please move it either to https://github.com/scrapinghub/splash/blob/master/docs/install.rst, or to a separate README file inside dockerfiles/splash?